test/index.spec.js

```diff
@@ -110,5 +111,36 @@ describe('immutableStateInvariantMiddleware', () => {
     expect(() => {
       dispatch({type: 'SOME_ACTION'});
     }).toNotThrow();
-  })
+  });
+
+  it('logs if mutating inside the dispatch', () => {
+    sinon.spy(console, 'error');
+    const next = action => {
+      state.foo.bar.push(5);
+      return action;
+    };
+
+    const dispatch = middleware({ logToConsoleOnly: true })(next);
+
+    expect(() => {
+      dispatch({type: 'SOME_ACTION'});
+    }).toNotThrow();
+    expect(console.error.calledWith(sinon.match.any, sinon.match(new RegExp('foo\\.bar\\.3')))).toEqual(true);
```

Comment:
Instead of `sinon.match.any`, I think we could use `console` directly, since that's what we're calling it with (and would fail if called otherwise).

Reply:
Can you clarify this suggestion by elaborating a bit more? Thanks.

Reply:
Ah sorry I got confused thinking the first parameter expected was the context (`this`), but it's not. Ok yeah we can keep the `any` here, since we don't really care about this here.